epochtalk/epochtalk
# Private boards

Sets the minimum view level for boards and categories. View-ability is determined by role priority.

Testing Procedures:
- Go to the management board page
- Pick a category and assign it a view level (Super administrator)
- Ensure that only that priority and above can see it.
- Do the same for boards
- For more fun, mix the view levels of categories and boards. (Categories should always have priority over the boards they have.)
